ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/ExpressionTypeManager.java

```diff
@@ -87,6 +88,29 @@
   private final LogicalSchema schema;
   private final FunctionRegistry functionRegistry;
 
+
+  private static final class ExpressionTypeContext {
```

Comment:
nit: you can just use a simple name like `Context` here. It's a common convention for the internal context passed around in `Visitor`s